web/client/src/components/request/ReqModule.jsx

```diff
@@ -0,0 +1,269 @@
+import React, { useState } from "react";
+import { Formik, Form, Field, FieldArray } from "formik";
+import {
+  Select,
+  CircularProgress,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  TextField,
+  Typography,
+  Button,
+  Grid,
+  OutlinedInput,
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  FormHelperText,
+  IconButton,
+} from "@material-ui/core";
+import AddCircleOutlineIcon from "@material-ui/icons/AddCircleOutline";
+import DeleteIcon from "@material-ui/icons/Delete";
+import "../../styles/request.css";
+import { object, string } from "yup";
+
+/**
+ * @return {Formik} component using Formik for creating send Request
+ */
+export default function ReqModule() {
+  const [open1, setOpen1] = useState(false);
+  const [open2, setOpen2] = useState(false);
+
+  const handleClickOpen = (num) => {
+    if (num === 1) {
+      setOpen1(true);
+    } else if (num === 2) {
+      setOpen2(true);
+    }
+  };
+
+  const handleClose = (num) => {
+    if (num === 1) {
+      setOpen1(false);
+    } else if (num === 2) {
+      setOpen2(false);
+    }
+  };
```

Comment:
I don't think you need to have a handleClickOpen or a handleClose function. You can just call onClick = {() => setOpen1(true)}. A similar thing could be done for the other button by calling setOpen2.

Reply:
Fixed, but idk why you can't see the commit on the PR